### ISSUE TITLE
Add support for random and static routing for remoting services

### ISSUE THREAD

**Issue by eliaslindholm** [AvanzaBank/astrix#4]
Example code:

```java
public interface MyService {
     // Invocations are randomly routed
    @RandomRouting
    void myMethod();
}

```

```java
public interface MyService {

    // Inocations are always routed using "1" as routing key
    @StaticRouting(1)
    void myMethod();

    // Static routing key defaults to 1
    @StaticRouting
    void myMethod();
}
```

**Comment by rogfor** [comment]
Cool

On 6 aug 2015, at 16:18, Elias Lindholm <notifications@github.com<mailto:notifications@github.com>> wrote:


Example code:

public interface MyService {
     // Invocations are randomly routed
    @RandomRouting
    void myMethod();
}


public interface MyService {

    // Inocations are always routed using "1" as routing key
    @StaticRouting(1)
    void myMethod();

    // Static routing key defaults to 1
    @StaticRouting
    void myMethod();
}

**Comment by eliaslindholm** [comment]
Released 0.34.1
